docs/tutorials/other/xmtp-metamask-snap.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_label: MetaMask Snap
+sidebar_position: 3
+---
+
+# Use the "Sign in with XMTP" MetaMask Snap
+
+Use the “Sign in with XMTP” MetaMask Snap to securely and conveniently sign in to any web app built with XMTP.
+
+When you install the Snap, you give it permission to store your XMTP user keys in MetaMask secure storage. From then on, when you use an app you’ve authorized to work with Sign in with XMTP, the Snap seamlessly and securely allows the app to use those keys without accessing the key material directly, enabling you to start messaging without needing to provide a signature.
+
+## FAQ about Sign in with XMTP
+
+### How do I install Sign in with XMTP?
+
+Use [MetaMask](https://metamask.io/) to sign in to any web app built with XMTP that supports MetaMask Snaps. [View supported apps](https://github.com/xmtp/snap)
+
+- If this is your first time using an app built with XMTP, you’ll be prompted to sign with your wallet account to create and enable an XMTP identity and generate your user keys. You’ll then be given the option to install Sign in with XMTP and store your user keys in MetaMask secure storage.
```

Comment:
I realized I gave you the wrong order. It's install first then sign/enable.

Reply:
Updated!